### PR TITLE
Adding UI changes for Unity driver

### PR DESCRIPTION
**Pull request by chaganti-rajitha**
# Description
Adding Unity driver to Installation Wizard UI.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/698 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [] Have you tested whether the hyperlinks are working properly?
- [] Did you add the examples wherever applicable?
- [] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csminstallationwizard/src/index.html [hunk 1]
```diff
@@ -331,6 +343,18 @@
                     </div>
                   </div>
 
+                  <div class="was-validated row mb-4 fsGroupPolicy">
+                    <label for="fsGroupPolicy" class="col-sm-2 col-form-label" data-bs-toggle="tooltip" data-bs-placement="right" title="Select the fsGroupPolicy type" style="width: 180px;">FSGroupPolicy Type</label>
+                    <div class="col-sm-9"></div>
+                    <div class="col-sm-3">
+                      <select id="fsGroup-Policy" class="form-select" onchange="validateInput(validateForm, CONSTANTS)" required>
+                        <option value="" selected> Select the FSGroupPolicy type</option>
+                        <option value="ReadWriteOnceWithFSType">ReadWriteOnceWithFSType</option>
```
Comment (niranjan-n1) [inline content/docs/deployment/csminstallationwizard/src/index.html:352]: Can we make ReadWriteOnceWithFSType to be selected by default if it is the default value?

**Review by rsedlock1958 — COMMENTED** [review]
No grammatical edits required

File: content/docs/deployment/csminstallationwizard/src/index.html (hunk 1 above)
Comment (chaganti-rajitha) [reply content/docs/deployment/csminstallationwizard/src/index.html]: Updated the code, Please check

**Review by niranjan-n1 — APPROVED** [review]
LGTM

**Review by samihan-dell — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**